CMake/SlicerDirectories.cmake

```diff
@@ -51,12 +51,18 @@ if(NOT DEFINED Slicer_MAIN_PROJECT_APPLICATION_NAME)
   set(Slicer_MAIN_PROJECT_APPLICATION_NAME ${${Slicer_MAIN_PROJECT}_APPLICATION_NAME})
 endif()
 
+# This opens the possibility to define alternative lib directories
+# (e,g., lib64/ comonly used in contemporary GNU/Linux sytems).
```

Comment:
```suggestion
# (e.g., lib64/ commonly used in contemporary GNU/Linux systems).
```

Reply:
@jamesobutler, thanks for having a look.